thanksgiving/thanksgiving_assign.py

```diff
@@ -0,0 +1,12 @@
+import datetime as dt
+
+def days_until_thanksgiving():
```

Comment:
Great idea here, but I think what you'll want to do is start from the last day of November and find the first Thursday that occurs before that. Your solution works most of the time but might fail if, for instance, November 1st is a Thursday.